apps/trading/components/rewards-container/Rewards.graphql

```diff
@@ -92,3 +186,28 @@ query RewardsEpoch {
     id
   }
 }
+
+query VestingDetails($partyId: ID!) {
+  party(id: $partyId) {
```

Comment:
Why not just add to vestingStats in the RewardsPage query

Reply:
fair enough, updated